server/services/xcpService.ts

```diff
@@ -763,6 +763,29 @@ export class XcpManager {
     }
   }
 
+  static async getTxInfo(tx: string): Promise<number | null> {
```

Comment:
this doesn't really fit within the xcpService file since it's not really related to xcp calls. probably lib/utils be suitable to go along with similar utxoUtils and broadcast.ts. 

should also just utilize the BLOCKSTREAM_API_BASE_URL var in constants.

It looks like it willl just return a null value if it fails connecting to blockstream (blocked by usage limits, etc) - this should fallback to N/A for the value to the client